handlers/pom.xml

```diff
@@ -34,6 +34,7 @@
     <module>handler-bizkeeper</module>
     <module>handler-flowcontrol-qps</module>
     <module>handler-loadbalance</module>
+    <module>handler-auth</module>
```

Comment:
there are so many "auth" types, do we need to rename to a concrete name?

Reply:
accept